tools/packaging/guest-image/lib_se.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
```

Comment:
didn't review this file. I assume it's just a copy without modification ;)

Reply:
The only change I made was to relocate `rm -f "${parmfile}"` from a trap callback to the end of the `build_secure_image()` function. Thanks!